platform-emf/src/main/java/com/softicar/platform/emf/test/tester/EmfFormPopupTester.java

```diff
@@ -70,6 +71,7 @@ public void clickCheckbox(IDbBooleanField<?> booleanField) {
 	public DomTab findTab(IDisplayString tabName) {
 
 		return findNode(DomTabBar.class)//
+			.findNode(DomTabMarker.TAB_BAR_HEADER)
```

Comment:
why do we need this marker? cant you just use this?
```suggestion
		return findNode(DomTabBarHeader.class)//
```